R/options.R

```diff
@@ -0,0 +1,88 @@
+#' Get or set Xportr options
+#'
+#' @description
+#'
+#' There are two mechanisms for working with options for xportr. One is the
+#' [options()] function, which is part of base R, and the other is the
+#' `xportr_options()` function, which is in the xportr package. The reason for
+#' these two mechanisms is has to do with legacy code and scoping.
+#'
+#' The [options()] function sets options globally, for the duration of the R
+#' process. The [getOption()] function retrieves the value of an option. All
+#' xportr related options of this type are prefixed with `"xportr."`.
+#'
+#'
+#' @section Options with `options()`:
+#'
+#' \describe{
+#' \item{xportr.df_domain_name}{defaults to `"dataset"`}:
+#'  The name of the domain "name" column in dataset metadata.
+#' \item{xportr.df_label}{defaults to `"label"`}:
+#'  The column noting the dataset label in dataset metadata.
+#' \item{xportr.domain_name}{defaults to `"dataset"`}:
+#'  The name of the domain "name" column in variable metadata.
```

Comment:
Is this duplicated?

Reply:
They seem to be different. One is `xportr.domain_name` while the other is `xportr.df_domain_name`
I made sure to follow the same order for these options as we had in the [xportr package](https://github.com/atorus-research/xportr/blob/bcf4c88d1e4b0a900410c309993ee23c3b2111fe/R/xportr-package.R#L12-L77) docs which was not easily visible before.

Reply:
ah yes - apologies!! you are correct.